fuel-core/src/database/storage.rs

```diff
@@ -0,0 +1,48 @@
+use fuel_core_interfaces::{
+    common::{
+        fuel_crypto::fuel_types::Bytes32,
+        fuel_storage::Mappable,
+        fuel_tx::{
+            Receipt,
+            UtxoId,
+        },
+    },
+    model::FuelBlockDb,
+};
+use fuel_txpool::types::ContractId;
+
+/// The table of blocks generated by Fuels validators.
+/// Right now, we have only that type of block, but we will support others in the future.
```

Comment:
> Right now, we have only that type of block, but we will support others in the future.

Not sure what this comment means, are you talking about sealed blocks?

Reply:
We discussed that in the future, we plan to have the blocks from ethereum. But I'm not sure=) I made this assumption based on the current naming `FuelBlock`.

Reply:
we won't be storing actual eth blocks though, just the metadata like validator staking diffs or messages like we already do.

Reply:
Then, maybe we can rename `FuelBlock` to `Block`?=) Because it is already in the `fuel-core`, it belongs to Fuel.

Reply:
Yeah this is probably fine. Separate PR though.